pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.netcracker.educationcenter</groupId>
     <artifactId>elasticsearch</artifactId>
-    <version>1.0</version>
```

Comment:
Usually X.Y.Z versioning is used or just X.Y
X - major changes
Y - minor changes
Z - patch (fix bugs)
check https://en.wikipedia.org/wiki/Software_versioning